HomeWork/src/main/java/io/FileLoggerConfiguration.java

```diff
@@ -0,0 +1,105 @@
+package io;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class FileLoggerConfiguration {
+
+    String fileName = "NIO.txt";
+    private LoggingLevel loggingLevel;
+    private LoggingLevel[] arrayLevels;
+    protected int maxSizeFile = 1024;
+    private String formatFile = " [ПОТОЧНИЙ_ЧАС][LEVEL] ";
+
+    protected File logFile = new File(fileName);
```

Comment:
remove this field, it is redundant as you're already creating a file in the `createAdditionalFile()` method.